### ISSUE TITLE
[Bug]: Duplicate gas fee fiat estimate

### ISSUE THREAD

**Issue by PeterYinusa** [MetaMask/metamask-extension#17397]
### Describe the bug

The fiat fee is displayed twice

<img width="1616" alt="gas display duplicate value" src="https://user-images.githubusercontent.com/53189696/214428641-2fdb27e5-a441-43be-8c62-0ace39b170b1.png">

### Steps to reproduce

1. Login
2. Open the account menu
3. Click `Settings` menu item
4. Click `General` sub menu item
5. Change the `Primary Currency` to `Fiat` 
6. Click `Advanced` sub menu item
7. Turn on `Show conversion on test networks`
8. Navigate back to the overview page and click the `Send` button
9. Enter an address

### Version

10.25

### Build type

None

### Browser

Chrome

### Operating system

MacOS

**Comment by PeterYinusa** [comment]
@digiwand is this something you can look at, seems related to https://github.com/MetaMask/metamask-extension/pull/17207

**Comment by digiwand** [comment]
Hey @PeterYinusa! It does seem a bit related

The first time I repro'd the steps, I saw the issue you saw. I think the value in the second row still represents the fiat conversation of the gas estimate above - which is expected. I think the thing, in this case, is the `maxFee` matched the estimated fee. I'm not sure why. Could this be a bug or a coincidental occurrence? I checked this by changing the gas fee value to see all values updated.

The second time I repro'd the steps, I saw distinct values:
https://user-images.githubusercontent.com/20778143/215146716-46fe1e55-2af2-4e05-bc27-aa4e9fe69cf5.mov
